### DISCUSSION TITLE
Rust version

### DISCUSSION THREAD

**Discussion by kanekoshoyu** [eclipse-iceoryx/iceoryx2#71] [Q&A]
Is there a specific reason why rust version 1.72.1 is used?

In my own setup the iceoryx2 is working well, but at my server it's using rust version 1.70.0, and it complained that this crate requires version 1.72.1+.

**Comment by elBoberido** [comment]
@kanekoshoyu not sure about that. It seems to build fine with 1.70.0 and the tests also pass. Maybe @elfenpiff has more insight. It might also just be that 1.72.1 was the stable Rust version during the 0.1 release.

**Reply by elfenpiff** [reply]
@kanekoshoyu I think we can reduce the version to 1.70.0, I will add this as an issue and to the upcoming version `0.2.0`, see: https://github.com/eclipse-iceoryx/iceoryx2/issues/72
